OpenSim/Simulation/Test/testForces.cpp

```diff
@@ -782,6 +774,131 @@ void testFunctionBasedBushingForce()
 
     ASSERT(*copyOfSpring == spring);
 }
+
+void testExpressionBasedBushingForce()
+{
+    using namespace SimTK;
+    
+    double mass = 1;
+    double stiffness = 10;
+    double start_h = 0.5;
+    double ball_radius = 0.25;
+    
+    double omega = sqrt(stiffness/mass);
+    
+    double dh = mass*gravity_vec(1)/stiffness;
+    
+    // Setup OpenSim model
+    Model *osimModel = new Model;
+    osimModel->setName("ExpressionBasedBushingTest");
+    osimModel->setGravity(gravity_vec);
+
+    // Create ball body and attach it to ground
+    // with a vertical slider
+
+    const Ground& ground = osimModel->getGround();
+
+    OpenSim::Body ball("ball", mass, Vec3(0), mass*SimTK::Inertia::sphere(0.1));
+    ball.addMeshGeometry("sphere.vtp");
+    ball.scale(Vec3(ball_radius), false);
+    
+    SliderJoint sliderY("", ground, Vec3(0), Vec3(0,0,Pi/2), ball, Vec3(0), Vec3(0,0,Pi/2));
+    
+    double positionRange[2] = {-10, 10};
+    // Rename coordinates for a slider joint
+    CoordinateSet &slider_coords = sliderY.upd_CoordinateSet();
+    slider_coords[0].setName("ball_h");
+    slider_coords[0].setRange(positionRange);
+    slider_coords[0].setMotionType(Coordinate::Translational);
+    
+    osimModel->addBody(&ball);
+    osimModel->addJoint(&sliderY);
+    
+    // Create base body and attach it to ground with a weld
+
+    OpenSim::Body base("base_body", mass, Vec3(0), mass*SimTK::Inertia::sphere(0.1));
+    base.addMeshGeometry("sphere.vtp");
+    base.scale(Vec3(ball_radius), false);
+    
+    WeldJoint weld("", ground, Vec3(0), Vec3(0), base, Vec3(0), Vec3(0));
+    osimModel->addBody(&base);
+    osimModel->addJoint(&weld);
+    
+    // create an ExpressionBasedBushingForce that represents an
+    // uncoupled, linear bushing between the ball body and welded base body
+    Vec3 rotStiffness(0);
+    Vec3 transStiffness(stiffness);
+    Vec3 rotDamping(0);
+    Vec3 transDamping(0);
+    
+    ExpressionBasedBushingForce spring("base_body", Vec3(0), Vec3(0), "ball", Vec3(0), Vec3(0), transStiffness, rotStiffness, transDamping, rotDamping);
```

Comment:
Violation of [80 character line limit](https://github.com/opensim-org/opensim-core/blob/master/CONTRIBUTING.md#each-line-of-text-should-be-at-most-80-characters) :rabbit2: